release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -3658,4 +3658,33 @@ $ oc adm release info 4.10.43 --pullspecs
 [id="ocp-4-10-43-upgrading"]
 ==== Updating
 
-To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
\ No newline at end of file
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-44"]
+=== RHBA-2022:8702 - {product-title} 4.10.44 bug fix update
+
+Issued: 2022-12-06
+
+{product-title} release 4.10.44 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:8702[RHBA-2022:8702] advisory. There are no RPM packages for this release.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.10.44 --pullspecs
+----
+
+[id="ocp-4-10-44-enhancements"]
+==== Enhancements
+
+* IPv6 unsolicited neighbor advertisements and IPv4 gratuitous address resolution protocol now default on the SR-IOV CNI plug-in. Pods created with the Single Root I/O Virtualization (SR-IOV) CNI plug-in, where the IP address management CNI plug-in has assigned IPs, now send IPv6 unsolicited neighbor advertisements and/or IPv4 gratuitous address resolution protocol by default onto the network. This enhancement notifies hosts of the new pod's MAC address for a particular IP to refresh ARP/NDP caches with the correct information. For more information, see xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported devices].
+
+[id="ocp-4-10-44-bug-fixes"]
+==== Bug fixes
+
+* Previously, object storage instances responded with `204 No Content` when there was no content to display. The {rh-openstack-first} SDK used in {product-title} did not handle 204s correctly. With this update, the installation program works around the issue when there are zero items to list. (link:https://issues.redhat.com/browse/OCPBUGS-4160[OCPBUGS-4160])
+
+[id="ocp-4-10-44-upgrading"]
+==== Updating
+
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
```

Comment:
```suggestion
To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
```

Reply:
I did not see a section with the title 'Updating a cluster within a minor version by using the CLI'. I might have missed it though!